README.md

```diff
@@ -1,2 +1,18 @@
-# federation
-A federation scheduler for multi-cluster
+# volcano-global
+
+volcano-global is a multi-cluster task scheduling application
```

Comment:
Change to `volcano-global is a multi-cluster scheduler, designed to efficiently manage and schedule AI/ML, Bigdata workloads across multiple kubernetes clusters, which cleverly combines the capabilities of [Karmada](https://karmada.io/) and [Volcano](https://volcano.sh/en/) ` is better.